### DISCUSSION TITLE
webUI: Unable to load cluster list，What went wrong?

### DISCUSSION THREAD

**Discussion by xaserver** [rackslab/Slurm-web#339] [Q&A] [answered]
**WEB:**

Unable to load cluster list
Try to refresh…

**Browser Tips：**
ERROR
Server error: Request error: Network Error

Rocky linux 9.3
slurmctld -V
slurm 23.11.6
systemctl status slurm-web-agent.service
● slurm-web-agent.service - Slurm-web HPC dashboard agent
     Loaded: loaded (/usr/lib/systemd/system/slurm-web-agent.service; enabled; preset: disabled)
     Active: active (running) since Wed 2024-08-21 22:43:36 CST; 7min ago
   Main PID: 811 (slurm-web-agent)
      Tasks: 1 (limit: 100186)
     Memory: 47.3M
        CPU: 39ms
     CGroup: /system.slice/slurm-web-agent.service
             └─811 /usr/bin/python3 -s /usr/libexec/slurm-web/slurm-web-agent

8月 21 22:43:36 master systemd[1]: Started Slurm-web HPC dashboard agent.
8月 21 22:43:38 master slurm-web-agent[811]: MainThread: [WARNING] Caching is disabled
8月 21 22:43:38 master slurm-web-agent[811]: MainThread: [INFO] Running slurm-web-agent application
8月 21 22:43:38 master slurm-web-agent[811]:  * Serving Flask app 'slurm-web-agent' (lazy loading)
8月 21 22:43:38 master slurm-web-agent[811]:  * Environment: production
8月 21 22:43:38 master slurm-web-agent[811]:    WARNING: This is a development server. Do not use it in a production deployment.
8月 21 22:43:38 master slurm-web-agent[811]:    Use a production WSGI server instead.
8月 21 22:43:38 master slurm-web-agent[811]:  * Debug mode: off
8月 21 22:43:38 master slurm-web-agent[811]: MainThread: [INFO]  * Running on http://localhost:5012/ (Press CTRL+

[root@master ~]# systemctl status slurm-web-gateway.service 
● slurm-web-gateway.service - Slurm-web HPC dashboard gateway
     Loaded: loaded (/usr/lib/systemd/system/slurm-web-gateway.service; enabled; preset: disabled)
     Active: active (running) since Wed 2024-08-21 22:43:36 CST; 9min ago
   Main PID: 812 (slurm-web-gatew)
      Tasks: 1 (limit: 100186)
     Memory: 48.2M
        CPU: 128ms
     CGroup: /system.slice/slurm-web-gateway.service
             └─812 /usr/bin/python3 -s /usr/libexec/slurm-web/slurm-web-gateway

8月 21 22:52:05 master slurm-web-gateway[812]: Thread-43: [INFO] 127.0.0.1 - - [21/Aug/2024 22:52:05] "GET /assets/index-db70270e.css HTTP/1.1" 304 -
8月 21 22:52:05 master slurm-web-gateway[812]: Thread-42: [INFO] 127.0.0.1 - - [21/Aug/2024 22:52:05] "GET /assets/index-45f37282.js HTTP/1.1" 304 -
8月 21 22:52:05 master slurm-web-gateway[812]: Thread-44: [INFO] 127.0.0.1 - - [21/Aug/2024 22:52:05] "GET /config.json HTTP/1.1" 200 -
8月 21 22:52:05 master slurm-web-gateway[812]: Thread-45: [INFO] 127.0.0.1 - - [21/Aug/2024 22:52:05] "OPTIONS /api/anonymous HTTP/1.1" 200 -
8月 21 22:52:21 master slurm-web-gateway[812]: Thread-46: [INFO] 10.10.8.160 - - [21/Aug/2024 22:52:21] "GET /anonymous HTTP/1.1" 200 -
8月 21 22:52:21 master slurm-web-gateway[812]: Thread-47: [INFO] 10.10.8.160 - - [21/Aug/2024 22:52:21] "GET /assets/index-45f37282.js HTTP/1.1" 200 -
8月 21 22:52:21 master slurm-web-gateway[812]: Thread-48: [INFO] 10.10.8.160 - - [21/Aug/2024 22:52:21] "GET /assets/index-db70270e.css HTTP/1.1" 200 -
8月 21 22:52:22 master slurm-web-gateway[812]: Thread-49: [INFO] 10.10.8.160 - - [21/Aug/2024 22:52:22] "GET /config.json HTTP/1.1" 200 -
8月 21 22:52:22 master slurm-web-gateway[812]: Thread-50: [INFO] 10.10.8.160 - - [21/Aug/2024 22:52:22] "GET /favicon.ico HTTP/1.1" 200 -
8月 21 22:52:22 master slurm-web-gateway[812]: Thread-51: [INFO] 127.0.0.1 - - [21/Aug/2024 22:52:22] "OPTIONS /api/anonymous HTTP/1.1" 200 -

[root@master ~]# systemctl status slurmrestd.service 
● slurmrestd.service - Slurm REST daemon
     Loaded: loaded (/usr/lib/systemd/system/slurmrestd.service; enabled; preset: disabled)
    Drop-In: /etc/systemd/system/slurmrestd.service.d
             └─slurm-web.conf
     Active: active (running) since Wed 2024-08-21 22:43:37 CST; 10min ago
   Main PID: 1254 (slurmrestd)
      Tasks: 21 (limit: 100186)
     Memory: 10.3M
        CPU: 0
     CGroup: /system.slice/slurmrestd.service
             └─1254 /usr/sbin/slurmrestd unix:/run/slurmrestd/slurmrestd.socket

8月 21 22:43:37 master systemd[1]: Started Slurm REST daemon.

**Comment by xaserver** [comment]
![1724394933826](https://github.com/user-attachments/assets/362d8a36-c75b-4087-9868-d789d92a00bd)

**Comment by rezib** [comment, marked as answer]
There should be a typo somewhere in your config (`localhost` _vs_ `loaclhost`).
